### PR TITLE
[Blog entry] Journey with Airflow as an Outreachy intern

### PR DESCRIPTION
**Pull request by OmairK**

### CODE REVIEW

**Comment by mik-laj** [comment]
@mschichensoup Can I ask for review?

**Comment by mik-laj** [comment]
@mschickensoup, Can I ask for review?

File: landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[Outreachy](https://www.outreachy.org/) is a program which organises three months paid internships with FOSS
+projects for people who are typically underrepresented in those projects.
+
+The first thing I had to do was choose a project under an organisation. After going through all the projects
+I chose “Extending the REST API of Apache Airflow”, because I had a good idea of REST API(s) so I thought it
+would be easier to get started with the contributions. I had never contributed to FOSS before so this part
+was overwhelming but there were plenty of issues labelled “good first issues” with detailed descriptions and
+some even had code snippets so luckily that nudged me in the direction of my first PR.
+
+My first PR made me familiar with the feedback loop and those feedbacks on my subsequent PRs were the focal
```
Comment (leahecole) [inline landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md:20]: grammar nit - "those feedbacks" -> the feedback

File: landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+point of the overall learning experience. Through actively and passively participating in discussions I
+learnt that even if there is a difference of opinion I could always learn from the different approaches.
+Throughout my internship I learnt a lot about different frameworks and technologies but the biggest takeaway
+for me was that a code is read more often than its written, and I started writing code with that in mind.
```
Comment (leahecole) [inline landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md:24]: Love this!! So true.

File: landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md (hunk 1 above)
Comment (leahecole) [inline landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md:20]: Consider linking to your PR on "my first PR" - I bet folks would love to see it! Additionally, consider listing your additional ones somewhere - not only will it give users some good examples of healthy PR discussions, it'll be a nice reference for you in the future :)

Comment (OmairK) [reply landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md:20]: That sounds fantastic I have made this PR WIP for now. I want to add some more lines to this blog, I kinda skimped on some incidents related to my first PR :sweat_smile:.

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Should we merge @OmairK ? Or do you plan to add more content ?

**Comment by OmairK** [comment]
> Should we merge @OmairK ? Or do you plan to add more content ?



> Should we merge @OmairK ? Or do you plan to add more content ?

@potiuk I just remembered something I wanted to add. I will ping you for the review, sorry for any trouble.

File: landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Through actively and passively participating in discussions I learnt that even if there is a difference of opinion
+one could always learn from the different approaches, and [this PR](https://github.com/apache/airflow/pull/8721) with
+more than 300+ comments is the proof of it. I also started reviewing small PRs which gave me the amazing opportunity
+to interact with new people. Throughout my internship I learnt a lot about different frameworks and technologies
+but the biggest takeaway for me was that a code is read more often than it's written, and I started writing code with
+that in mind.
+
+### Wrapping Up
+So with my project of extending Airflow’s REST API as well as the Outreachy internship coming to an end I would like
+to thank my mentors [Jarek Poutik](https://github.com/potiuk), [Kaxil Naik](https://github.com/kaxil) and
```
Comment (potiuk) [inline landing-pages/site/content/en/blog/experience-with-airflow-as-an-outreachy-intern.md:57]:
```suggestion
to thank my mentors [Jarek Potiuk](https://github.com/potiuk), [Kaxil Naik](https://github.com/kaxil) and
```

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Love it. Merging !

**Pull request merged**